tests/foreman/ui/test_bookmark.py

```diff
@@ -14,8 +14,10 @@
 from robottelo.test import UITestCase
 from robottelo.ui.locators import common_locators, locators
 from robottelo.ui.session import Session
+from unittest2 import skipIf
 
 
+@skipIf(bz_bug_is_open(1322003), 'Skipping due to open Bugzilla bug #1322003')
```

Comment:
You can apply that to setUp method.

Using skipIf will call the bugzilla check code that will make an HTTP request which could lead in unexpected lock issues. The reason why the robottelo decorators runs at run time is to avoid that.